.circleci/config.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       - run:
           name: install requirements
           command: |
-            python3 -m venv venv
+            python -m venv venv
```

Comment:
This should be `python3` but you have to invalidate the cache.
I see that the CI configuration is not very reliable. I suggest you check this configuration file to see how to setup a test matrix: https://github.com/allo-media/eventail/blob/master/.circleci/config.yml